python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -297,10 +300,15 @@ def _apply_default_remote_args(ray_remote_args: Dict[str, Any]) -> Dict[str, Any
 class _MapWorker:
     """An actor worker for MapOperator."""
 
-    def __init__(self, ctx: DatasetContext, src_fn_name: str):
+    def __init__(
+        self, ctx: DatasetContext, src_fn_name: str, init_fn: _CallableClassProtocol
```

Comment:
```suggestion
        self, ctx: DatasetContext, src_fn_name: str, init_fn: Callable[[], None]
```

This should be the narrower interface?

Reply:
good point, updated